modules/developers-guide/pages/tutorials/calculator.adoc

```diff
@@ -238,15 +238,15 @@ include::example$candid-ui.adoc[]
 +
 image:candid-calc.png[Calculator functions]
 
-== Stop the local network
+== Stop the local canister execution environment
 
-After you finish experimenting with your program, you can stop the local Internet Computer network so that it doesn’t continue running in the background.
+After you finish experimenting with the program, you can stop the local canister execution environment  so that it doesn't continue running in the background.
```

Comment:
```suggestion
After you finish experimenting with the dapp, you can stop the local canister execution environment  so that it does not continue running in the background.
```